.circleci/config.yml

```diff
@@ -51,7 +51,10 @@ jobs:
       - image: circleci/ruby:2.6
     steps:
       - checkout
-      - run: "mkdir -p ~/test-results/rspec && mkdir -p ~/test-results/rubocop"
+      - run:
+          name: Create test-results dir
+          command: mkdir -p ~/test-results/rspec
```

Comment:
Not sure if this is actually an issue or not, but should we also create the `~/test-results/linting` folder that we use in row 76?

Reply:
My initial attempt added yard-junk as its own job separate from the rubocop one. However, it [failed](https://circleci.com/gh/lostisland/faraday/142?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) because `~/test-results/yard` doesn't exist, even though I [explicitly created it](https://github.com/lostisland/faraday/pull/958/commits/e503819a38d1f6ff8870369e9cd1df5d549e283b).

From further testing, `~/test-results/rubocop` doesn't exist either for the currrent `linting` job. It looks like Rubocop takes the initiative in creating the empty directories.

Though, perhaps instead of relying on Rubocop to take the initiative, the `linting` job should handle its own directory creation as an initial `run` step?

Reply:
That's fine, except they're not being persisted outside of the `checkout_code` job's container. It's only persisting `.` (which is `~/project`), so `~/test-results` is skipped. 

I've updated this to create the test-results dirs in the same container that they're used.